### DISCUSSION TITLE
Dexie to query JavaScript arrays

### DISCUSSION THREAD

**Discussion by sherazakbar50** [dexie/Dexie.js#1606] [Q&A]
Hi, I am working on an application in angular that is making great use of IndexedDb via dexie. I am in love with dexie due to its support to query indexed db stores in a sql like way. My question is can we use WhereClause of dexie with simple javascript arrays along with indexeddb stores?  What i meant is 
[{Id:1,test:"ABC"},{Id:2,test:"XYZ"}].where("test").equals('ABC').toArray()

**Comment by naton** [comment]
You mean something like `Object.values([{ Id: 1, test: "ABC" },{ Id: 2, test: "XYZ" }].find(obj => obj.test === 'ABC'))`?
